proposals/blittable.md

```diff
@@ -8,129 +8,97 @@
 
-``` c#
-blittable struct Point 
-{
-    public int X;
-    public int Y;
-}
-```
+- Have the type `sbyte`, `byte`, `short`, `ushort`, `int`, `uint`, `long`, `ulong`, `char`, `float`, `double`, `decimal`, `bool`, `IntPtr` or `UIntPtr`.
+- Be any `enum` type.
+- Be a pointer type.
+- Be a user defined struct that satsifies the `unmanaged` constraint.
 
-The compiler will enforce that the fields of such a struct definition fit one of the following categories:
+Compiler generated instance fields, such as those backing auto-implemented properties, must also meet these constraints. 
 
-- `sbyte`, `byte`, `short`, `ushort`, `int`, `uint`, `long`, `ulong`, `char`, `float`, `double`, `decimal`, or `bool`.
-- Any `enum` type
-- Any pointer type which points to a `blittable` type
-- Any user defined struct explicitly declared as `blittable`
+For example:
 
-Any type fitting the definition above, or any element in the list, is considered a valid `blittable` type.
+``` c# 
+// Unmanaged type
+struct Point { 
+    int X;
+    int Y {get; set;}
+}
 
-``` c#
-struct User
-{
+// Not an unmanaged type
+struct Student { 
     string FirstName;
     string LastName;
 }
+``` 
 
-blittable struct ItemData
-{
-    // Error: blittable struct cannot contain field of type User which is not blittable
-    User User;
-    int Id;
-}
-```
+The `unmanaged` constraint cannot be combined with `struct`, `class` or `new()`. This restriction derives from the fact that `unmanaged` implies `struct` hence the other constraints do not make sense.
 
-Note that a user defined struct must be explicitly declared as `blittable` in order to meet the requirements above.  This is required even if the struct otherwise meets the requirements of `blittable`. 
+The `unmanaged` constraint is not enforced by CLR, only by the language. To prevent mis-use by other languages, methods which have this constraint will be protected by a mod-req. This will 
+prevent other languages from using type arguments which are not unmanaged types.
 
-``` c#
-struct SimplePoint
-{
-    public int X;
-    public int Y;
-}
-
-blittable struct Data
-{
-    // Error: blittable struct cannot contain field of type SimplePoint which is not blittable
-    SimplePoint Point;
-}
-```
-
-The language will also support the ability to constrain generic type parameters to be `blittable` types.  
-
-``` C#
-void M<T>(T p) where T : blittable struct
-{
-    ...
-}
-
-M<Point>(); // Ok
-M<User>();  // Error: Type User does not satisfy the blittable constraint.
-```
-
-One of the primary motivations for `blittable` structs is ease of interop.  As such it's imperative that such a type have it's field ordered in a sequential, or explicit, layout.  An auto layout of fields makes it impossible to reliably interop the data.  
+The token `unmanaged` in the constraint is not a keyword, nor a contextual keyword. Instead it is like `var` in that it is evaluated at that location and will either:
```

Comment:
pedantry: it is worth calling out if @unmanaged or other escapes are allowed in this identifier. or if it needs to be the exact text 'unamanged'.  You do mention that it works like 'var' (which does not allow '@var'), but it would be worth being explicit.